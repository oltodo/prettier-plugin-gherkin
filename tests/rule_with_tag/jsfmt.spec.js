run_spec(__dirname, ["gherkin"]);
