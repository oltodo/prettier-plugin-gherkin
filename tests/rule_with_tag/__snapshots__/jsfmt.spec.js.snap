// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rule_with_tag.feature - gherkin-verify: rule_with_tag.feature 1`] = `
@tag_feature
Feature: Some tagged rules

  Rule: Untagged rule
    The untagged rule description

    Scenario: Scenario with only a feature tag
      Given a

  @tag_rule
  Rule: Tagged rule
    The tagged rule description

    Scenario: Scenario with feature and rule tags
      Given b

    @tag_scenario
    Scenario: Scenario with feature, rule and scenario tags
      Given b
`;
